packages/browser/src/sessionInfo/SessionInfoManager.ts

```diff
@@ -33,14 +33,15 @@ import {
 } from "@inrupt/solid-client-authn-core";
 import { v4 } from "uuid";
 import { clearOidcPersistentStorage } from "@inrupt/oidc-client-ext";
+import { fetchWithCookies } from "../ClientAuthentication";
 
 export function getUnauthenticatedSession(): ISessionInfo & {
   fetch: typeof fetch;
 } {
   return {
     isLoggedIn: false,
     sessionId: v4(),
-    fetch,
+    fetch: fetchWithCookies,
```

Comment:
Just checking if `fetchWithCookies` is what you want here; the original one was a regular fetch.

Reply:
The previous workaround that prevented the bug from manifesting (wrongfully removed in https://github.com/inrupt/solid-client-authn-js/commit/520a19bdf9ca2d4e6e7e377a3dcf24053cf4ba70#diff-4f6320deb9e43b5216428fb7c2dd230f4b0c5325c6f147f6a73b15659c355fb9L132, I must have messed up testing then) did include the cookie, so that you are still able to access private resources on refresh. This is a stopgap solution that only works against ESS, and has known limitations, which is why we will replace it as soon as possible with a proper fix to cleanly resolve the refresh issue in a way that's independent f the resource server, and inline with the OIDC philosophy.